.travis.yml

```diff
@@ -169,19 +169,11 @@ matrix:
         - bundle exec pod install
 
     - language: swift
-      name: "Swift - SPM (Xcode 10.2)"
+      name: "Swift - SPM (Xcode 11.1)"
       os: osx
-      osx_image: xcode10.2
-      cache:
-        directories:
-          - .build
-      before_install:
-        - gem update --system
-        - gem install bundler
-        - swift --version
-        - swift build --verbose
+      osx_image: xcode11.1
       script:
-        - swift test
+        - xcodebuild -scheme "Workflow-Package" test -destination "name=iPhone 11"
```

Comment:
SPM generates a scheme called "<PackageName>-Package" to disambiguate if the package name is the same as one of its products (as is the case here)